VSharp.Explorer/Explorer.fs

```diff
@@ -94,207 +105,265 @@ type private SVMExplorer(explorationOptions: ExplorationOptions, statistics: SVM
         | _ -> false
 
     let rec mkForwardSearcher mode =
-        let getRandomSeedOption() = if options.randomSeed < 0 then None else Some options.randomSeed
+        let getRandomSeedOption () =
+            if options.randomSeed < 0 then
+                None
+            else
+                Some options.randomSeed
         match mode with
         | AIMode ->
             match options.aiAgentTrainingOptions with
             | Some aiOptions ->
-                match aiOptions.oracle with 
-                | Some oracle -> AISearcher(oracle, options.aiAgentTrainingOptions) :> IForwardSearcher
-                | None -> failwith "Empty oracle for AI searcher."           
+                match aiOptions.oracle with
+                | Some oracle -> AISearcher (oracle, options.aiAgentTrainingOptions) :> IForwardSearcher
+                | None -> failwith "Empty oracle for AI searcher."
             | None ->
                 match options.pathToModel with
-                | Some s -> AISearcher s
+                | Some s ->
+                    let useGPU =
+                        if options.useGPU.IsSome then
```

Comment:
It can be written as single boolean expression because of lazy boolean expression evaluation
```
let useGPU = options.useGPU.IsSome &&  options.useGPU.Value
```

Reply:
Solved